src/pages/ProductList/ProductList.scss

```
@@ -0,0 +1,112 @@
+.productList {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+  height: 50%;
+
+  .productListContainer {
+    display: flex;
+    justify-content: space-around;
+
+    overflow-x: scroll; //레이아웃 승인 완료 되면 지울것임
+    white-space: nowrap;
+    .productContainer {
+      position: relative;
+      margin: 5px 10px;
+      transform: translate(630px);
+      .productImg {
+        width: 359px;
+        height: 359px;
+        img {
+          width: 100%;
+          height: 100%;
+        }
```

```suggestion
    .productContainer {
      position: relative;
      margin: 5px 10px;
      transform: translate(630px);
      
      .productImg {
        width: 359px;
        height: 359px;
        
        img {
          width: 100%;
          height: 100%;
        }
```

스타일 구역도 가독성을 위해 스타일마다 한칸씩 띄워주세요!